docs/modules/ROOT/pages/extensions/activemq.adoc

```diff
@@ -0,0 +1,27 @@
+// Do not edit directly!
+// This file was generated by camel-quarkus-package-maven-plugin:update-extension-doc-page
+
+[[activemq]]
+= ActiveMQ
+
+*Since Camel Quarkus 1.0.0-M5, JVM and native*
```

Comment:
Might be better to separate the support level (JVM and native) out into it's own heading. WDYT?

Reply:
Yes, I am not 100% satisfied with this part too. I'd personally prefer some sort of badges visually similar to these 
![image](https://user-images.githubusercontent.com/1826249/82461803-1891ed80-9abb-11ea-8089-cdc57d3f08b2.png)
I'd try to find some antora/css expert once the first iteration is online.